desk/lib/hooks-json.hoon

```diff
@@ -93,11 +93,11 @@
       %gone  (id id.r)
       %order  (order-rsp +.r)
       %config  (config-rsp +.r)
-      %wait  (wait-rsp +.r)
+      %cron  (cron-rsp +.r)
       %rest  (rest-rsp +.r)
     ==
   ++  set-rsp
```

Comment:
If by convention we rename hook response to `$r-hook`, this function and others below would be adjusted accordingly
```suggestion
  ++  r-set
```

Reply:
will leave for later